src/core/config.rs

```diff
@@ -79,16 +79,14 @@ pub struct ConfigDefaults {
 
 impl ConfigDefaults {
     /// Open and deserialize a defaults file
-    pub fn read(file: &str) -> LalResult<ConfigDefaults> {
-        let pth = Path::new(file);
-        if !pth.exists() {
-            error!("No such defaults file '{}'", file); // file open will fail below
+    pub fn new() -> Self {
+        ConfigDefaults {
+            backend: BackendConfiguration::Local(LocalConfig{}),
+            environments: BTreeMap::<String, Environment>::new(),
+            mounts: Vec::<Mount>::new(),
+            minimum_lal: Option::<String>::None,
+
```

Comment:
this could almost be an `impl Default for Config` now if we restructure a bit, but otherwise this is good!
EDIT: nm, artifactory is default :/